profiles/admin.py

```diff
@@ -177,6 +179,7 @@ class ExtendedProfileAdmin(VersionAdmin):
     ]
     change_list_template = "admin/profiles/profiles_changelist.html"
     list_filter = ("service_connections__service",)
+    raw_id_fields = ("user",)
```

Comment:
What about `autocomplete_fields` ? :thinking:

Reply:
Louhis you are a genius! Why didn't I know of that. 😄